x-pack/test/api_integration/deployment_agnostic/apis/observability/apm/diagnostics/data_streams.spec.ts

```diff
@@ -45,9 +43,13 @@ export default function ApiTest({ getService }: FtrProviderContext) {
     });
 
     describe('When data is ingested', () => {
+      let apmSynthtraceEsClient: ApmSynthtraceEsClient;
+
       before(async () => {
-        const latestVersion = await synthtraceKibanaClient.fetchLatestApmPackageVersion();
-        await synthtraceKibanaClient.installApmPackage(latestVersion);
+        apmSynthtraceEsClient = await synthtrace.createApmSynthtraceEsClient();
+        const apmSynthtraceKibanaClient = synthtrace.apmSynthtraceKibanaClient;
+        const latestVersion = await apmSynthtraceKibanaClient.fetchLatestApmPackageVersion();
+        await apmSynthtraceKibanaClient.installApmPackage(latestVersion);
```

Comment:
`createApmSynthtraceEsClient` already installs the latest apm package version. Why is this needed?

Reply:
Thanks for review. Those were remainings from the old test and I didn't notice that apm packge is installed in `createApmSynthtraceEsClient`. Updated!